examples/reg-issue/src/utils/typography.js

```diff
@@ -3,16 +3,16 @@ import Typography from 'typography'
 const typography = new Typography({
   googleFonts: [
     {
-      name: 'Raleway',
-      styles: ['400', '800'],
+      name: 'Buenard',
+      styles: ['700'],
     },
     {
-      name: 'Source Sans Pro',
-      styles: ['300'],
-    },
+      name: 'Inconsolata',
+      styles: ['400'],
+    }
   ],
-  headerFontFamily: ['Raleway', 'Helvetica Neue', 'Helvetica', 'sans-serif'],
-  bodyFontFamily: ['Source Sans Pro', 'Helvetica', 'sans-serif'],
+  headerFontFamily: ['Buenard', 'Helvetica Neue', 'Helvetica', 'sans-serif'],
```

Comment:
let's update the fallback fonts here. Buenard is serif and Inconsolata is monospace iirc

Reply:
we probably don't need any other fallbacks than those two, just because the web fonts should load